src/tests/test_pyodide.py

```diff
@@ -1757,22 +1757,21 @@ def helper(selenium):
 )
 def test_runpython_filename(selenium, run_python):
     msg = selenium.run_js(
-        """
-        try {
-            %s(`
+        f"""
+        try {{
+            {run_python}(`
```

Comment:
Can we revert these changes to `test_pyodide.py` and `test_streams.py`? I'd rather use `%s` than have to double all the curly braces. f strings aren't that good for templating JS.

Reply:
Ok, but I'm not sure how to ignore those with ruff.

Reply:
Okay I disabled UP038 in pyproject.toml

Reply:
Ah, thanks !